beast/tests/test_regresscheck.py

```diff
@@ -453,25 +452,18 @@ def test_read_noise_data(self):
         for ckey in ndata.keys():
             assert ckey in exp_keys, f"{ckey} not in noise data expected keys"
 
-        # check an entry for a single model (caching current values 18 Apr 2020)
-        # fmt: off
-        exp_bias = [-6.77602149e-20, -1.36353610e-20, 2.87448605e-20,
-                    -2.38253474e-21, -1.70330281e-20, -2.70390708e-20]
-        exp_error = [1.63128160e-19, 7.50503350e-20, 7.65873857e-20,
-                     2.48842055e-20, 9.41313147e-20, 2.79650823e-20]
-        exp_compl = [1.0, 0.95552407, 1.0, 0.74733078, 0.77777778, 0.42857143]
-        # fmt: on
-        np.testing.assert_allclose(
-            ndata["bias"][10], exp_bias, err_msg="Expected bias values not correct",
-        )
-        np.testing.assert_allclose(
-            ndata["error"][10], exp_error, err_msg="Expected error values not correct",
-        )
-        np.testing.assert_allclose(
-            ndata["completeness"][10],
-            exp_compl,
-            err_msg="Expected completeness values not correct",
-        )
+        # check that the values are reasonable
```

Comment:
Will we fill these in with real values later?  I could imagine that a change to the code would yield reasonable results, but still different, which we'd want to know about.

Reply:
They are reasonable.  I just forgot to remove the comment that was related to actual values I had in the test previously.  The actual values can change now, but they have to be reasonable.  Makes the test not need to be revised every time something upstream is changed.  Trying to make the tests more robust.